src/utils/breachResolution.test.js

```diff
@@ -147,13 +149,22 @@ test('appendBreachResolutionChecklist: data class with a resolution referring to
     unverifiedEmails: []
   }
   appendBreachResolutionChecklist(userBreachData)
-  // There should only be a resolution for `BreachDataTypes.Phone`, as
-  // `BreachDataTypes.Passwords` and `BreachDataTypes.SecurityQuestions` refer
-  // to the breached company's domain, which we don't know:
+  // There should be a resolution for `BreachDataTypes.Phone`,
+  // `BreachDataTypes.Passwords` and `BreachDataTypes.SecurityQuestions`.
+  // The last two should fallback to a more generic header string that does not
+  // include the breached company's domain, which we don't know:
```

Comment:
I don't know how easy it is to mock AppConstants, but if it's easy (but only then - you've been working on this PR for long enough), maybe a test for the blocklist would be a good addition?

Reply:
Good point! I’ll address this in a follow-up in order to not block this PR.